bin/cmd.js

```diff
@@ -17,15 +15,15 @@ const getStdin = require('get-stdin')
 /**
  * @param {Omit<import('../').LinterOptions, 'cmd'> & StandardCliOptions} rawOpts
  */
-function Cli (rawOpts) {
+function cli (rawOpts) {
   const opts = {
     cmd: 'standard-engine',
     tagline: 'JavaScript Custom Style',
     version: '0.0.0',
     ...rawOpts
   }
 
-  const Linter = require('../').linter
+  const { Linter } = require('../')
   const standard = rawOpts.linter || new Linter(opts)
```

Comment:
The intention with doing  `require('../')` here rather than in the top of the file must have been to avoid loading it if another linter had been provided, but then the code would have to be:

```suggestion
  const standard = rawOpts.linter || new ( require('../').Linter)(opts)
```

Just an observation 🤔

Reply:
Somehow, if I put `const { Linter } = require('../')` at the top of the file, the tests are failing :
`TypeError: Linter is not a constructor`.
Do you have an idea, why?
Maybe you can test it yourself locally (I'm using Node.js v16.9.1).

But, yes if I take your recommendation `const standard = rawOpts.linter || new (require('../').Linter)(opts)`, it is working, I'll change it then. :+1: